pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -38,7 +38,10 @@
 
 class ModelCheckpoint(Callback):
     r"""
-    Save the model after every epoch by monitoring a quantity.
+    Save the model after every epoch by monitoring a quantity. Every metric logged with
+    :meth:`~pytorch_lightning.core.lightning.log` or :meth:`~pytorch_lightning.core.lightning.log_dict` in
+    LightningModule is a candidate for the monitor key. For more information, see
+    :ref:`common/weights_loading:Checkpoint saving`.
```

Comment:
not 100% sure, but I think we can only reference sections by title within the same document. 
You probably have to refer to just :ref:`_weights_loading`